docs/Product/DataAnalysisValidation.mdx

```diff
@@ -1,27 +1,29 @@
 ---
 sidebar_position: 10.11
-sidebar_label: "Data analysis and validation"
+sidebar_label: "Data Analysis and Validation"
```

Comment:
Please leave Data analysis and validation. As highlighted here, we should use _sentence case_ for titles https://open-energy-transition.github.io/handbook/docs/Handbook/HandbookStyleGuide